tests/unit/core/test_store.js

```diff
@@ -3,6 +3,7 @@ import { middleware } from 'core/store';
 describe('core store middleware', () => {
   function configForDev(isDevelopment, config = {}) {
     const finalConfig = { isDevelopment, ...config };
+
```

Comment:
I know this is just old code but `configForDev` can use [`getFakeConfig()`](https://github.com/mozilla/addons-frontend/blob/master/tests/unit/helpers.js#L461) now.